js/src/stackutil.ts

```diff
@@ -8,7 +8,7 @@ export interface StackTraceEntry {
 
 export function getStackTrace(): StackTraceEntry[] {
   const trace = new Error().stack;
-  if (trace === undefined) {
+  if (typeof trace !== "string") {
     return [];
   }
   const traceLines = trace.split("\n");
```

Comment:
@akulkapoor how about we coerce here:

```ts
String(trace).split('\n')
```

worst case we have an useless trace, and best case we notice what the weird trace is